README.adoc

```diff
@@ -120,7 +120,9 @@ incubator  https://github.com/appsody/stacks/releases/latest/download/incubator-
 ----
 
 *Recommendation*: In enterprise settings, when a solution architect creates application stacks with
-technology choices that are in a private Collection Hub, it's best to remove `incubator` from the list:
+technology choices that are in a private Collection Hub, it's best to remove `incubator` from the list.
+These Appsody stacks are not supported by the Kabanero application cluster. Run the following command
+to remove the `incubator` repository:
```

Comment:
Maybe also mention that `When the collections are built on the cluster using the default pipeline, it will fail if the collection pointed to in the appsody project in the .appsody-config.yaml is not in the list of collections activated on the cluster.`

Reply:
Hmm - I understand the intention here, but I think this is too much of a sudden deep-dive for Jane, who doesn't need to understand the complexity of the stack she's building on. I wonder whether this information needs to go into Champs "Working with collections" guide, although the hidden files are not explained there either. I guess `.appsody-config.yaml` is hidden for a reason.

Reply:
Yeah this check is sort of the intersection of Jane, Todd, and Champ's universe.  Let's go with what you had initially and will work on maybe improving the error message to provide additional hints.